client/graphql/types/index.tsx

```diff
@@ -336,6 +339,13 @@ export type QueryCertificationArgs = {
 };
 
 
+export type QueryCertificationsByProviderNameArgs = {
```

Comment:
This file contains excess types, no query named `CertificationsByProviderName` exists on the repo, yet. Please regenerate.